### PR TITLE
Pagination and other things

### LINKED ISSUE #3: Pagination
Can you add pagination to this theme?

Thank you.

### LINKED ISSUE #6: Markdown Highlighting
Really love this theme but I cannot get any markdown styling or highlighting to work.  I.e. headings and code syntax.  Could you possibly point me in the right direction to get this working, or possibly it is working and if how could I change it to standard github md styling?

Cheers,
Justin
[wagonhelm.github.io](http://wagonhelm.github.io)

### PR DESCRIPTION
**Pull request by jpasholk**
Hey man!

Sorry, but I got carried away and made a few other changes that I thought were needed! If you'd like I can split those out into another pull request. :)

[Here](https://jpasholk.com/Soot-Spirits/) a link to a working demo of my changes.

A quick summary is:

* Added pagination (obviously) which would close #3 

* Fixed lists from being misaligned, I have a screenshot if you'd like to see it, but Bulma was making them have 0 margin and 0 padding. 

* Added a gemfile for managing dependances.

* Added a .gitignore file.

* Added syntax highlighting which might close #6 

* Fixed headings - they weren't being styled. (was this intended?)

* Added ```relative_url``` to a lot of links so they'd work everywhere. 

* A few other minor things I can't remember, lol. 

Anyway, let me know if you have any questions!

:)

Josh

### CODE REVIEW

**Comment by jpasholk** [comment]
I just realized that my version isn’t pulling the quotes in the header for some reason... any idea why?

**Comment by jpasholk** [comment]
Okay, I fixed it... It wasn't pulling the file because I'm serving it from a subdirectory. :) That shouldn't affect this pull.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,30 @@
-# Soot Spirits Jekyll Theme
+# My Fork of the Soot Spirits Jekyll Theme
```
Comment (abhn) [inline README.md]: Can we skip edits to the readme? We can have a `CHANGELOG` file to log the changes if we must.

**Comment by abhn** [comment]
Hey buddy @jpasholk 

Thank you for your efforts. It looks good now.

I have added a little comment for the readme file. About demo, I'll push these changes to the other organization where this theme is hosted (sootspirits.github.io). Except that, I think it is good to go. 

Also, we might want to add a contributors section where I can give you credits. I'll do it for you.

Thanks again :-)
Abhishek

File: README.md (hunk 1 above)
Comment (jpasholk) [reply README.md]: Hey! That shouldn’t be in the branch that I submitted the pull request from. Is it?

But yeah, we can keep the readme as you’d like. :)

**Comment by jpasholk** [comment]
Right on man! Did you want me to change the Readme back? I thought I was only changing it on my master branch to serve the demo for pagination. :)

**Comment by abhn** [comment]
Yeah, no problem at all. Let's revert it. 

Changes should go into a separate file, but no need of that for now.

Also, just a general feedback for your other open source adventures :) One PR should only address one problem. That makes it easier for the maintainers to review the code and provide pointed feedback (to help us learn). While not a real problem here as this is a tiny project, it will help in larger OSS.

Thanks again.

**Comment by jpasholk** [comment]
Okay, I’ll revert the the README and place the change log in a different file.

I seem to be having some issues with my internet connection this morning, so it may take me some time. :)

Thanks for the feedback, I really should have made separate pull requests for the other stuff! Sorry about that. 😅

**Comment by abhn** [comment]
So worries. Whenever you find time. Cheers!

**Comment by jpasholk** [comment]
Okay! I think I've got it all ready. Just had to power cycle my router and modem. Thanks again, I had a wonderful time yesterday figuring this stuff out, although I did get carried away, I'll know for next time to keep things simple.

Cheers!

**Review by abhn — APPROVED** [review]

**Comment by abhn** [comment]
Thanks! :)

**Pull request merged**